src/aws-redshiftserverless/workgroup.ts

```diff
@@ -0,0 +1,357 @@
+import { IResource, Lazy, Names, Resource, aws_redshiftserverless, aws_ec2, Token, Stack } from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+import { INamespace } from './namespace';
+
+/**
+ * A Redshift Serverless Workgroup
+ */
+export interface IWorkgroup extends IResource, aws_ec2.IConnectable {
+  /**
+   * The workgroup Arn
+   *
+   * @attribute
+   */
+  readonly workgroupArn: string;
+  /**
+   * The workgroup name
+   *
+   * @attribute
+   */
+  readonly workgroupName: string;
+  /**
+   * The workgroup id
+   *
+   * @attribute
+   */
+  readonly workgroupId: string;
+  /**
+   * The workgroup endpoint address
+   *
+   * @attribute
+   */
+  readonly endpointAddress: string;
+  /**
+   * The workgroup port
+   *
+   * @attribute
+   */
+  readonly port: number;
+}
+
+/**
+ * Properties for defining a Redshift Serverless Workgroup.
+ */
+export interface WorkgroupProps {
+  /**
+   * The base compute capacity of the workgroup in Redshift Processing Units (RPUs).
+   *
+   * You can adjust the Base capacity setting from 8 RPUs to 512 RPUs in units of 8.
+   * Also you can increment or decrement RPUs in units of 32 when setting a base capacity between 512-1024.
+   *
+   * @default 128
+   *
+   * @see https://docs.aws.amazon.com/redshift/latest/mgmt/serverless-capacity.html
+   */
+  readonly baseCapacity?: number;
+
+  /**
+   * A list of parameters to set for finer control over a database.
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-redshiftserverless-workgroup.html#cfn-redshiftserverless-workgroup-configparameters
+   * @default - no config parameters
+   */
+  readonly configParameters?: { [key: string]: string };
+
+  /**
+   * The value that specifies whether to enable enhanced virtual private cloud (VPC) routing,
+   * which forces Amazon Redshift Serverless to route traffic through your VPC.
+   *
+   * @default - false
```

Comment:
If it is certain that this will happen

```suggestion
   * @default false
```